lms/static/lms/js/spec/main.js

```diff
@@ -679,6 +679,7 @@
     });
 
     testFiles = [
+        'course_experience/js/spec/course_outline_factory_spec.js',
```

Comment:
I agree that I won't make any changes here, but it is an interesting point. @bjacobel, do you have any ideas?